packages/gatsby-image/src/index.js

```diff
@@ -379,12 +370,22 @@ class Image extends React.Component {
                 backgroundColor: bgColor,
                 width: image.width,
                 opacity: !this.state.imgLoaded ? 1 : 0,
-                transitionDelay: `0.25s`,
+                transitionDelay: initialDelay,
                 height: image.height,
               }}
             />
           )}
 
+          {/* Show the blurry base64 image. */}
```

Comment:
what's the reason for moving it down?

Reply:
As the PR and commits should explain, it was to fix the layer ordering with `backgroundColor` placeholder, so that it can be used as a fallback with an image placeholder. Presently the image would be covered by the `backgroundColor` placeholder.